src/main/java/PalindromeChecker.java

```diff
@@ -3,11 +3,14 @@
 
     public boolean check(String word) {
 
+        //still needed?
         if(word.length() == 1) {
             return true;
         }
 
+        //reassigning? - just question-marks?
         word = word.toLowerCase().replaceAll(" ", "").replaceAll("\\?", "");
+        //charArray?
```

Comment:
I'm not sure why you convert your String to a char[] and then to a StringBuilder - can you make it simpler?